h5p-bildetema-words-grid-view/src/components/TopicWordsGrid/TopicWordsGrid.tsx

```diff
@@ -22,16 +22,19 @@ export const TopicWordsGrid: React.FC<TopicWordsGridProps> = ({
   const contentId = useContentId();
 
   React.useEffect(() => {
-    document
-      .getElementById(`toggle-${contentId}`)
-      ?.addEventListener("change", handleChange);
+    // Temp Hack: Navigating from topic -> words causes document.getElementById(`toggle-${contentId}`) to be null, if we not wait...
+    setTimeout(() => {
+      document
+        .getElementById(`toggle-${contentId}`)
+        ?.addEventListener("change", handleChange);
+    }, 10);
```

Comment:
Will it work to wait one frame? All rendering is supposed to be done by then.

```suggestion
    requestAnimationFrame(() => {
      document
        .getElementById(`toggle-${contentId}`)
        ?.addEventListener("change", handleChange);
    });
```

Reply:
Is there a reason why this is not handled by React?

Reply:
This was added because when it was handled by react and we toggled the showWrittenWords button it caused all the images to "blink". (guess they reloaded)